build-system/dep-check-config.js

```diff
@@ -148,6 +148,10 @@ exports.rules = [
   {
     filesMatching: '{src,extensions}/**/*.js',
     mustNotDependOn: '3p/**/*.js',
+    whitelist: [
+      'extensions/amp-analytics/0.1/transport.js->' +
+        '3p/iframe-messaging-client.js',
```

Comment:
This exception is added because the amp-analytics tag needs to send messages to the third-party iframe.